app/containers/TracksContainer/index.js

```diff
@@ -102,11 +102,15 @@ export function TracksContainer({
               ParentComponent={TrackGrid}
               renderItem={(item, index) => (
                 <TrackComponent
+                  isShowDetailsBtn
```

Comment:
either use abbreviated or fullforms and keep it consistent
isShowDetailsButton could be better . What do you think ?

Reply:
Updated accordingly